psga/config/psga-schema-sars-cov-2.yaml

```diff
@@ -204,56 +180,56 @@ pipeline:
         name: SARS CoV 2
         slug: sars-cov-2
 
-    # NEB-VarSkip_v1a
```

Comment:
Not sure why we don't want these primers as we support them..

Reply:
I can't remember why it broke, nothing stands out as being wrong about it and they should be supported. It seems we actually need to port all these things to the BE and see if it works.

Reply:
> > I don't understand this PR. It seems to me that the config in the BE is outdated, rather than the config in this repo. This should not be merged to me.
> 
> @pdp10 The backend has changes that the Pipeline doesn't have, and the Pipeline has changes that the backend doesn't have. This change synchronises the 2 to be identical.

I cannot see anything that this PR `adds` to the pipeline config, whereas I see a significant loss in functionality.